applications/CoSimulationApplication/custom_python/add_co_sim_io_to_python.cpp

```diff
@@ -170,44 +171,14 @@ void ImportDataSizeCheck(const std::size_t ExpectedSize, const std::size_t Impor
 
 void ExportData_ModelPart_Scalar(
     CoSimIO::Info& rInfo,
-    const ModelPart& rModelPart,
+    ModelPart& rModelPart,
     const Variable<double>& rVariable,
     const DataLocation DataLoc)
```

Comment:
this should be the one from the aux-model-part-utilites, can you please remove the definition from line 57?

Reply:
I didn't get this. In which file I have to delete the line number 57?
do u  mean I have to completely remove your old methods (definitions) and call the methods of Auxiliar_model_part_utilities??

Reply:
check my last commit, there I removed it already :)

Reply:
Ohh Understood. I removed line 57